projects/overlapping-long-subsessions.kp/transform.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+from utils import seconds_to_days, days_diff_from_strs
+from math import floor
+
+def long_to_wide(sqlContext, df, columns):
+    df.createOrReplaceTempView("temp")
+    return sqlContext.sql(
+    """
+         SELECT {0},
+         FIRST(normalized_channel) as normalized_channel,
+         COLLECT_LIST(subsession_start_date) as subsession_start_date,
+         COLLECT_LIST(subsession_length) as subsession_length
+         FROM
+         (
+           SELECT *
+           FROM temp
+           ORDER BY subsession_start_date
```

Comment:
Understood! is there a field like profileSubsessionCounter in main_summary? I'm looking at https://github.com/mozilla/telemetry-batch-view/blob/master/src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala currently.